dev/package.json

```diff
@@ -69,27 +69,30 @@
 		"vscode:prepublish": "npm i && npm run compile && npm run copy-strings && npm run copy-license && npm run copy-templates",
 		"compile": "tsc -p ./",
 		"watch": "tsc -watch -p ./",
-		"postinstall": "node ./node_modules/vscode/bin/install",
-		"test": "npm run compile && rm -rf ./out/test/pomfiles && cp -R ./src/test/pomfiles ./out/test/ && node ./node_modules/vscode/bin/test",
+		"test-setup": "rm -rf ./out/test/suites/pomfiles && cp -R ./src/test/pomfiles ./out/test/suites",
+		"test": "node out/test/RunTests.js",
+		"pretest": "npm run vscode:prepublish && rm -rf ./out/test/suites/pomfiles && cp -R ./src/test/pomfiles ./out/test/suites",
 		"copy-strings": "cp -v ./src/constants/Strings.json ./out/constants/",
 		"copy-templates": "cp -R ./src/templates ./out/",
 		"copy-license": "cp -v ../LICENSE ../README.md ."
 	},
 	"devDependencies": {
 		"@types/circular-json": "^0.4.0",
-		"@types/mocha": "^5.2.7",
-		"@types/node": "^12.0.4",
+		"@types/mocha": "^7.0.2",
+		"@types/node": "^13.9.2",
 		"@types/stack-trace": "0.0.29",
-		"tslint": "^5.17.0",
-		"typescript": "^3.5.2",
-		"vscode": "^1.1.34"
+		"@types/vscode": "^1.38.0",
```

Comment:
You need the version of @types/vscode to match the vscode engine version, so don't use `^` or it will silently pick up minor releases, which you don't want.

Reply:
Hi @tetchel , I'm looking at the helloworld-test-sample that they provided.  They have `"@types/vscode": "^1.32.0"` with the engine

```
"engines": {
		"vscode": "^1.32.0"
	}, 
```
WIll you use `~` ?

Reply:
`~` should be fine but I used just "1.38.0"

Reply:
Ok, I'll do the same